test/helper/clean.ts

```diff
@@ -93,7 +93,7 @@ const getOnePsy = (
     ...getAddress(),
     diploma: 'Psychologie clinique de la santé',
     phone: faker.phone.phoneNumber('0# ## ## ## ##'),
-    email: faker.internet.email(),
```

Comment:
what's the difference ?

Reply:
`faker.internet.email()` uses a real domain name and therefore can lead to a real email address
`faker.internet.exampleEmail()` uses `example.[suffix]` domain and ensure it's a fake one